conda/recipes/rapids-doc-env/meta.yaml

```diff
@@ -42,6 +42,7 @@ requirements:
     - nbsphinx {{ nbsphinx_version }}
     - numpydoc
     - pandoc {{ pandoc_version }}
+    - pydata-sphinx-theme
     - recommonmark
     - sphinx
     - sphinx_rtd_theme
```

Comment:
@ajschmidt8 I haven't removed `sphinx_rtd_theme` yet because all other libraries still use this theme.